client/cli/cli.go

```diff
@@ -139,10 +164,18 @@ func handleExit(dbp *proctl.DebuggedProcess, line *liner.State, status int) {
 		}
 	}
 
-	die(status, "Hope I was of service hunting your bug!")
+	t.die(status, "Hope I was of service hunting your bug!")
 }
 
-func die(status int, args ...interface{}) {
+type Terminator struct {
```

Comment:
And this is exactly where I wanted to go, after the initial implementation (I was going to change Terminator to Terminal). I'll update to Term.

Of course, the more you add, the more you get closer to reimplement my "cmd" package (github.com/gobs/cmd) :)